heartbeat/Squid.in

```diff
@@ -109,7 +109,7 @@ for a squid instance managed by this RA.
 <longdesc lang="en">
 This is a required parameter. This parameter specifies a port number
 for a squid instance managed by this RA. If plural ports are used,
-you must specifiy the only one of them.
+you must specify the only one of them.
```

Comment:
I guess we should also update to multiple instead of plural and remove the "the" before only.